src/components/MessageManager/MessageManager.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import "./MessageManager.css";
+import MessagingWindow from "../MessagingWindow/MessagingWindow";
+
+import Firebase from "firebase";
+
+class MessageManager extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      friends: [],
+      friendClicked: {},
+      user: this.props.user
+    };
+  }
+
+  componentDidMount() {
+    this.friendsRef = Firebase.database().ref(
+      "users/" + this.props.user + "/friends/"
+    ); //reference to friends
+    this.friendsRef.on("value", snapshot => {
+      this.setState(
+        {
+          friends: snapshot.val() //gives complete list of friends
+        },
+        () => {
+          let newClickedList = {};
```

Comment:
rename `newClickedList` to something like `newFriendClicked`. its not a list, so its a little misleading